### PR TITLE
DOC multiple vault mount points

### PR DESCRIPTION
**Pull request by NBardelot**
related: #29734

---

This PR adds a paragraph to the "Hashicorp Vault Secrets Backend" documentation page, in order to better illustrate how the #29734 improvement can be used, following the `vault.py` behaviour in relation to the mount_point variable and the *_path variables.

### CODE REVIEW

**Review by potiuk — APPROVED** [review]
Nice!

**Comment by potiuk** [comment]
(but needs some doc fixes)

File: docs/apache-airflow-providers-hashicorp/secrets-backends/hashicorp-vault.rst [hunk 1]
```diff
@@ -214,3 +214,31 @@ Add "verify": "absolute path to ca-certificate file"
...
+  (the string will be split using the separator ``/``, the first element will be the mount point, the remaining
+  elements will be the path to the secrets)
+* leave ``config_path`` as the empty string ``""``
+* if you use ``config_path``, each configuration item will need to be prefixed with the ``mount_point`` used for configs,
+  as ``"mount_point/path/to/the/config"`` (here again, the string will be split using the separator ``/``, 
+  the first element will be the mount point, the remaining elements will be the path to the configuration parameter)
+
+For example:
+
+.. code-block:: ini
```
Comment (Taragolis) [inline docs/apache-airflow-providers-hashicorp/secrets-backends/hashicorp-vault.rst:238]: Missing blank line int the beginning of the code block, see: [Literal blocks ](https://www.sphinx-doc.org/en/master/usage/restructuredtext/basics.html#literal-blocks)

```suggestion
.. code-block:: ini

```

File: docs/apache-airflow-providers-hashicorp/secrets-backends/hashicorp-vault.rst [hunk 2]
```diff
@@ -214,3 +214,31 @@ Add "verify": "absolute path to ca-certificate file"
...
+
+In order to do this, you will need to setup you configuration this way:
+
+* leave ``mount_point`` as JSON ``null``
+* if you use ``variables_path`` and/or ``connections_path``, set them as ``"mount_point/path/to/the/secrets"``
+  (the string will be split using the separator ``/``, the first element will be the mount point, the remaining
+  elements will be the path to the secrets)
+* leave ``config_path`` as the empty string ``""``
+* if you use ``config_path``, each configuration item will need to be prefixed with the ``mount_point`` used for configs,
+  as ``"mount_point/path/to/the/config"`` (here again, the string will be split using the separator ``/``,
```
Comment (Taragolis) [inline docs/apache-airflow-providers-hashicorp/secrets-backends/hashicorp-vault.rst]:
```suggestion
  as ``"mount_point/path/to/the/config"`` (here again, the string will be split using the separator ``/``,
```

**Review by Taragolis — APPROVED** [review]

**Pull request merged**